src/utils/request.js

```diff
@@ -30,6 +30,8 @@ async function request(uri, format, options = {}) {
     options.headers.set('Accept', 'text/html')
   }
 
+  options.headers.append('user-key', config('endpoint_key'));
```

Comment:
### 1

Have you tried what happens when the `endpoint_key` is not provided? The `getConfig` method throws a javascript error - which stops further execution of the scripts, so existing applications might break (as far as I can tell?).

https://github.com/StadGent/npm_package_opening-hours-widget/blob/100133fa322fe51e3c1f394662283e74c2957c8b/src/config.js#L54-L60

This header is should only be sent when an `endpoint_key` is provided, so adding an extra check might be a good idea.

### 2

Also, I've looked a litte further into `Headers.set` and `Headers.append` and I quote:
> The difference between set() and Headers.append is that if the specified header already exists and accepts multiple values, set() overwrites the existing value with the new one, whereas Headers.append appends the new value to the end of the set of values.
> ~ _[Headers.set()](https://developer.mozilla.org/en-US/docs/Web/API/Headers/set)_

Since we'll never provide more than 1 value for the `user-key` header, `Headers.set` is the more reasonable choice here.

```suggestion
  if (config('endpoint_key')) {
    options.headers.set('user-key', config('endpoint_key'));
  }
```